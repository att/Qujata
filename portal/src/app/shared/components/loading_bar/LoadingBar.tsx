import cn from 'classnames';
import styles from './LoadingBar.module.scss';

export interface LoadingBarProps {
  className?: string;
}

export const LoadingBar: React.FC<LoadingBarProps> = (props: LoadingBarProps) => (
  <div className={cn(props.className, styles.loading_bar_container)}>
    <div className={styles.loading_bar} />
  </div>
);
