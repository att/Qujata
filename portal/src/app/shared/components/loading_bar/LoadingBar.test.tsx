import { render, RenderResult } from '@testing-library/react';
import { LoadingBar, LoadingBarProps } from './LoadingBar';

describe('LoadingBar', () => {
  test('renders LoadingBar', () => {
    const props: LoadingBarProps = {
    };
    const { container }: RenderResult = render(<LoadingBar {...props} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
