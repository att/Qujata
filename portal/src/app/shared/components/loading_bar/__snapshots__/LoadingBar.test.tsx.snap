// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingBar renders LoadingBar 1`] = `
<div
  class="loading_bar_container"
>
  <div
    class="loading_bar"
  />
</div>
`;
